<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Authentification</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Desktop/GRAPHICS/UI/ic_real_name_authentication_527859.png</normaloff>../../../Desktop/GRAPHICS/UI/ic_real_name_authentication_527859.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-180</x>
      <y>0</y>
      <width>1200</width>
      <height>600</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Desktop/GRAPHICS/UI/turnover1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid  rgb(9, 26, 52); 
border-radius: 12px;
padding: 10px 15px;</string>
    </property>
    <property name="placeholderText">
     <string>Company Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(9, 26, 52);</string>
    </property>
    <property name="text">
     <string>Company Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 53, 82);
border-radius: 12px;
box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
box-shadow: 0 5px #666;
transform: translateY(4px) ;
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Desktop/GRAPHICS/UI/unlock.png</normaloff>../../../Desktop/GRAPHICS/UI/unlock.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(9, 26, 52);</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid  rgb(9, 26, 52); 
border-radius: 12px;
padding: 10px 15px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>xxx@yyy.zzz</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(9, 26, 52);</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid  rgb(9, 26, 52); 
border-radius: 12px;
padding: 10px 15px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>At least 8 characters</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Desktop/GRAPHICS/UI/back icon.png</normaloff>../../../Desktop/GRAPHICS/UI/back icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(9, 26, 52);</string>
    </property>
    <property name="text">
     <string>Confirm your Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid  rgb(9, 26, 52); 
border-radius: 12px;
padding: 10px 15px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>At least 8 characters</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_4</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>label_5</zorder>
   <zorder>lineEdit_4</zorder>
   <zorder>commandLinkButton</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit_5</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../.designer/testt.qrc"/>
 </resources>
 <connections/>
</ui>
